spec/controllers/create_or_change_user_requests_controller_spec.rb

```diff
@@ -28,7 +28,6 @@ def valid_change_user_request_params
           "email"=>"subject@digital.cabinet-office.gov.uk",
         },
         "action" => "change_user",
-        "tool_role" => "govt_form",
```

Comment:
@evilstreak I've now split this change out into d838255. The reason why it's necessary to remove that redundant field is that `ActiveModel::Model` throws a wobbly if trying to set a non-existent field.